Patch by Robert Scheck <robert@fedoraproject.org> for pdftk >= 3.3.1, which changes the build-time Java
classpaths to /usr/share/java in order to pick up bcprov.jar and commons-lang3.jar from the distribution.

--- pdftk-3.3.1/build.xml						2021-08-23 21:40:44.000000000 +0200
+++ pdftk-3.3.1/build.xml.classpath					2021-09-03 02:38:14.633539811 +0200
@@ -15,7 +15,7 @@
     <mkdir dir="build/classes"/>
     <javac srcdir="java" destdir="build/classes" source="1.8" target="1.8">
       <classpath>
-        <fileset dir="lib" includes="*.jar"/>
+        <fileset dir="/usr/share/java" includes="*.jar"/>
       </classpath>
     </javac>
     <copy todir="build/classes">
@@ -43,7 +43,7 @@
     <mkdir dir="build/test"/>
     <javac srcdir="test" destdir="build/test">
       <classpath>
-        <fileset dir="lib" includes="*.jar"/>
+        <fileset dir="/usr/share/java" includes="*.jar"/>
         <pathelement path="build/classes"/>
       </classpath>
     </javac>
